packages/server/lib/server.ts

```diff
@@ -86,184 +42,6 @@ if (NANGO_MIGRATE_AT_START === 'true') {
-web.route('/api/v1/sync/command').post(webAuth, syncController.syncCommand.bind(syncController));
-web.route('/api/v1/syncs').get(webAuth, syncController.getSyncs.bind(syncController));
-web.route('/api/v1/sync/:syncId/frequency').put(webAuth, syncController.updateFrequency.bind(syncController));
-web.route('/api/v1/flows').get(webAuth, flowController.getFlows.bind(syncController));
-web.route('/api/v1/flow/deploy/pre-built').post(webAuth, flowController.deployPreBuiltFlow.bind(flowController));
-web.route('/api/v1/flow/download').post(webAuth, flowController.downloadFlow.bind(flowController));
-web.route('/api/v1/flow/:id/disable').patch(webAuth, flowController.disableFlow.bind(flowController));
-web.route('/api/v1/flow/:id/enable').patch(webAuth, flowController.enableFlow.bind(flowController));
-web.route('/api/v1/flow/:flowName').get(webAuth, flowController.getFlow.bind(syncController));
-
-web.route('/api/v1/onboarding').get(webAuth, onboardingController.status.bind(onboardingController));
-web.route('/api/v1/onboarding').post(webAuth, onboardingController.create.bind(onboardingController));
-web.route('/api/v1/onboarding').put(webAuth, onboardingController.updateStatus.bind(onboardingController));
-web.route('/api/v1/onboarding/deploy').post(webAuth, onboardingController.deploy.bind(onboardingController));
-web.route('/api/v1/onboarding/sync-status').post(webAuth, onboardingController.checkSyncCompletion.bind(onboardingController));
-web.route('/api/v1/onboarding/action').post(webAuth, onboardingController.writeGithubIssue.bind(onboardingController));
-
-// Hosted signin
-if (!isCloud && !isEnterprise) {
-    web.route('/api/v1/basic').get(webAuth, (_: Request, res: Response) => {
-        res.status(200).send();
-    });
-}
-
-// -------
-// 404
-web.use('/api/*', (_req: Request, res: Response) => {
-    res.status(404).json({ error: { code: 'not_found' } });
-});
-
-app.use(web);
-
-// -------
-// Webapp assets, static files and build.
-const webappBuildPath = '../../../webapp/build';
-const staticSite = express.Router();
-staticSite.use('/assets', express.static(path.join(dirname(), webappBuildPath), { immutable: true, maxAge: '1y' }));
-staticSite.use(express.static(path.join(dirname(), webappBuildPath), { setHeaders: () => ({ 'Cache-Control': 'no-cache, private' }) }));
-staticSite.get('*', (_, res) => {
-    res.sendFile(path.join(dirname(), webappBuildPath, 'index.html'), { headers: { 'Cache-Control': 'no-cache, private' } });
-});
-app.use(staticSite);
-
-// -------
-// Error handling.
-app.use(async (e: any, req: Request, res: Response, _: any) => {
-    await errorManager.handleGenericError(e, req, res, tracer);
-});
-
-// -------
-// Websocket
-const server = http.createServer(app);
-const wss = new WebSocketServer({ server, path: getWebsocketsPath() });
-
-wss.on('connection', async (ws: WebSocket) => {
-    await publisher.subscribe(ws);
-});
-
```

Comment:
you said in the PR description that none of the changes are used yet but server/server.ts is modified though. Am I missing something?

Reply:
yeah I meant it's changes for e2e tests but it's not used in tests yet